pkg/controller/user-cluster-controller-manager/resources/reconciler.go

```diff
@@ -521,6 +544,17 @@ func (r *reconciler) reconcileConfigMaps(ctx context.Context, data reconcileData
 	if err := reconciling.ReconcileConfigMaps(ctx, creators, metav1.NamespaceSystem, r.Client); err != nil {
 		return fmt.Errorf("failed to reconcile ConfigMaps in kube-system namespace: %v", err)
 	}
+
+	if r.userClusterMLA.Monitoring {
+		creators = []reconciling.NamedConfigMapCreatorGetter{
+			userclusterprometheus.ConfigMapCreator(userclusterprometheus.Config{
+				MLAGatewayURL: fmt.Sprintf("http://%s:%d/api/v1/push", r.clusterURL.Hostname(), r.userClusterMLA.MLAGatewayPort),
```

Comment:
are you sure it's `http` but not `https`?

Reply:
Since the MLA stack is not deployed in Seed and this feature is disabled by default, there is no traffic be sending anyways :) 
We have an issue to make it https: https://github.com/kubermatic/kubermatic/issues/6490